docs/sidebars.ts

```diff
@@ -6,7 +6,7 @@ const CATEGORY_SECTION_HEADER_MIXIN: SidebarItemCategoryBase = {
   collapsible: false,
   collapsed: false,
   className: "sidebar-section-title",
-}
+};
```

Comment:
looks like most of the changes in this file are formatting. I wonder why this is happening